docs/usage/README.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Logging
+description: "Gardener logging for an effective Kubernetes landscape operation"
+url: /docs/guides/logging
+type: tutorial-page
+level: beginner
+category: Operation
+scope: operator
+publishdate: 2020-03-25
+tags: ["task","operations","logging"]
+---
+
+This guide is about Gardener Logging, how it is organized and how to use the dashboard for viewing the Kubernetes clusters log data.
+
+## Cluster level logging
+Log data is fundamental for the successful operation activities of Kubernetes landscapes. It is used for investigating problems and monitoring cluster activity. 
+
+Cluster level logging is the recommended way to collect and store log data for Kubernetes cluster resources. With cluster level logging the log data is externalized 
+in a logging backend where the log lifecycle management is independent from the lifecycle management of the Kubernetes resources.
+
+Cluster level logging is not available by default with [Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/logging/#cluster-level-logging-architectures) and consumers have to implement it in addition.
+Kubernetes project provides only basic logging capabilities via _kubectl logs_ where the kubelet keeps one terminated container with its logs. 
+When a pod is evicted from the node, all corresponding containers are also evicted, together with their logs. 
+This is why the default log storage solution is considerd shot-lived and not sufficient when one wants to operate properly a Kubernetes environment. 
+
+Gardener, as an advanced Kubernetes management solution, follows the general recommendations and offers a cluster level logging solution to ensure proper log storage for all managed Kubernetes resources. 
+The log management is setup when the new cluster is created.
+Log collection is organized using [fluent-bit](https://fluentbit.io). 
+Log storage and search is organized using [Loki](https://grafana.com/oss/loki). 
+Log visualization is available using [Grafana](hhttps://grafana.com/grafana) that is deployed with 
+predefined dashboard and  visualization for every shoot cluster.
+
+
+Kubernetes operators can benefit from different capabilities like accessing the logs for 
+already terminated containers, perform fast and sophisticated search for investigating long 
+lasting problems or recurring problems based on logs from a long period of time. 
+
+Below in this guide, you will find the procedure how to access the log data for your clsuters.
+
+## Procedure
+
+Use this procedure to view the log data of your Kubernetes cluster.
+
+### Accessing Grafana
+1. Open **Gardener dashboard** and navigate to the cluster which log data you want to view
+2. Hover your mouse over cluster name to open and view the details for the cluster. Find the **Logging and Monitoring** section and use the link and credentials to log-in to Grafana.
```

Comment:
I think it's called "Monitoring" now.
```suggestion
2. Hover your mouse over cluster name to open and view the details for the cluster. Find the ** Monitoring** section and use the link and credentials to log-in to Grafana.
```

Also, is hovering a mouse sufficient to navigate? Did you mean click the cluster name? 
If so, the pattern for describing navigation we try to follow is in the following form "On the Gardener dashboard, choose CLUSTERS > [YOUR-CLUSTER] > OVERVIEW > Monitoring and use the link and credentials there to login to Grafana "